src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title: Examples of high-performance CSS animations
+authors:
+  - rachelandrew
+description: |
+  Demonstrating how high performance techniques can create complex and beautiful animations.
+date: 2020-10-20
+tags:
+  - animations
+  - performance
+---
+
+In this post find out how some popular animations found on CodePen have been created.
+THese animations all use the performant techniques discussed in other articles in this section.
+
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations, and the [Animations Guide](/animations-guide) for practical tips.
+
+## Wizard loading animation
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Only CSS Loader - Wizard by Guilmain Dorian'
+}
+%}
+
+This loading animation is built entirely with CSS.
+The image plus all of the animation has been created in CSS and HTML,
+no images or JavaScript.
+To understand how it was created and how well it performs you can use Chrome DevTools.
+
+With the animation running, open the Performance tab in Chrome DevTools and record a few seconds of the animation.
+You should see in the Summary that the browser is not doing any Layout or Paint operations when running this animation.
+
+<figure class="w-figure">
+  <img src="./wizard-summary.jpg" alt="Summary in DevTools">
+  <figcaption class="w-figcaption">
+    The summary after profiling the wizard animation.
+  </figcaption>
+</figure>
+
+To find out how this animation was achieved without causing layout and paint,
+inspect any of the moving elements in Chrome DevTools.
+For example select the triangle,
+and watch how the box of the element transforms during its journey into the air,
+as it spins, and then returns to the start position.
```

Comment:
Animation Inspector seems like an easier way to inspect? In that screenshot I'm hovering over `div.square` and DevTools is highlighting it in the viewport. Clicking it jumps me to that element in the DOM Tree.

![image](https://user-images.githubusercontent.com/4713486/96823936-79af4b80-13e2-11eb-9683-c67b1fb03960.png)

Reply:
We could show that - it's actually more clicks rather than just selecting the element but it's another tool to show.